app-shells/scsh/scsh-0.6.7-r3.ebuild

```diff
@@ -1,47 +1,56 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 inherit multilib
 
 MY_PV="${PV%*.*}"
 
 DESCRIPTION="Unix shell embedded in Scheme"
-HOMEPAGE="http://www.scsh.net/"
+HOMEPAGE="https://www.scsh.net/"
 SRC_URI="ftp://ftp.scsh.net/pub/scsh/${MY_PV}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 ppc sparc x86"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
```

Comment:
https://github.com/gentoo/gentoo/pull/22703#discussion_r749617041

Reply:
Sam's point is that you can add -r3 without dropping the stable -r2

Reply:
> Sam's point is that you can add -r3 without dropping the stable -r2

but there is no way that -r2 could have been installed on any system because of the AR bug

Reply:
> > Sam's point is that you can add -r3 without dropping the stable -r2
> 
> but there is no way that -r2 could have been installed on any system because of the AR bug

I see your point but someone did stabilize this ebuild. And before that another developer added this ebuild. Chances of two developers okaying this despite a clear build failure is very low I think. Unless, the two were the same developer, which appears to be the case here unfortunately ... You are likely to be right in this case, though it's simply safer to leave the stable version alone and revbump to unstable.